src/raylib.h

```diff
@@ -1546,6 +1563,7 @@ RLAPI void DrawModelEx(Model model, Vector3 position, Vector3 rotationAxis, floa
 RLAPI void DrawModelWires(Model model, Vector3 position, float scale, Color tint);          // Draw a model wires (with texture if set)
 RLAPI void DrawModelWiresEx(Model model, Vector3 position, Vector3 rotationAxis, float rotationAngle, Vector3 scale, Color tint); // Draw a model wires (with texture if set) with extended parameters
 RLAPI void DrawBoundingBox(BoundingBox box, Color color);                                   // Draw bounding box (wires)
+RLAPI void DrawTexturePro3D(Texture2D texture, Rectangle source, Rectangle3D dest, Vector2 origin, float rotation, Color tint); // Draw a part of a texture with 'pro' parameters in 3D space
```

Comment:
> I think this function could be renamed to `DrawTextureQuad()` and does not need to follow their 2D equivalent:
> 
> ```c
> DrawTextureQuad(Texture2D texture, Rectangle source, Transform transform, Color tint);
> ```
> 
> It's up to the user to provide transform data.

@raysan5 The function definition you provided seems unable to specify the center of rotation, and therefore cannot cover the existing functionality of `DrawBillboardPro`. In fact, I added this function as an underlying implementation for `DrawBillboardPro`, while also facilitating the creation of more functions for drawing simple textures in 3D space, such as non-camera-facing "billboards". In previous versions of Raylib, one had to use `DrawTexturePro` along with RLGL's matrix stack to directly draw textures in 3D space. If you prefer to avoid adding new structures, I might lean towards inlining this function within `DrawBillboardPro`.

Reply:
I see, in that case I prefer to inline this function into `DrawBillboardPro()` for now. Users with more advance/specific 3d requirements can still rely on `rlgl` functionality.